src/components/ConfirmedRoute.js

```diff
@@ -0,0 +1,119 @@
+import React, {useEffect} from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import MapView from 'react-native-x-maps';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import _ from 'underscore';
+import ONYXKEYS from '../ONYXKEYS';
+import CONST from '../CONST';
+import * as MapboxToken from '../libs/actions/MapboxToken';
+import * as Expensicons from './Icon/Expensicons';
+import theme from '../styles/themes/default';
+import styles from '../styles/styles';
+import transactionPropTypes from './transactionPropTypes';
+import BlockingView from './BlockingViews/BlockingView';
+import useNetwork from '../hooks/useNetwork';
+import useLocalize from '../hooks/useLocalize';
+
+const MAP_PADDING = 50;
```

Comment:
Why 50? Can this be cleaned up at all? It looks like it's a similar story in `DistanceRequest.js`. If we're always passing `50` then let's move this value to CONST